README.md

```diff
@@ -276,6 +304,10 @@ Compose.prototype.map = function(f) {
 
 #### `sequence` method
 
+```hs
+sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
```

Comment:
Should this be `Applicative f` rather than `Apply f`? `c -> f c` is `of` essentially, isn't it?